sur/library.hoon

```diff
@@ -0,0 +1,50 @@
+/-  *resource, store=graph-store
+|%
++$  comment  tape
++$  book
+  $:
+    title=tape
+    isbn=tape
+    :: TODO fulltext=tape ? 
+  ==
+::
+::  TODO better / clearer name for prim
++$  prim     (jug resource atom)  :: for a given resource, what indexes does the reader want to hear about?
```

Comment:
add a comment that this is a networking section

Reply:
https://github.com/ynx0/library/pull/3/commits/e0e660826361bf85fd8beda15a64217c9fc82b35